charms/tensorboard-controller/metadata.yaml

```diff
@@ -4,6 +4,9 @@ name: tensorboard-controller
 description: Kubeflow Tensorboard Controller
 summary: Kubeflow Tensorboard Controller
 series: [kubernetes]
+deployment:
+  type: stateless
+  service: omit
```

Comment:
Why are we adding this in both of the charms? These weren't showing these issues

Reply:
Removed those additions